storage/innobase/include/os0file.h

```diff
@@ -508,6 +515,9 @@ struct Encryption {
 
 	/** Current uuid of server instance */
 	static char		uuid[ENCRYPTION_SERVER_UUID_LEN + 1];
+
+	/** current maximum version, use to rencrypt all tables */
```

Comment:
```suggestion
/** current maximum version, used to avoid writing V3 encryption on old versions that support only upto V2 header */
```
ensure the length is < 80 chars and clang-formatted.